packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -108,14 +109,22 @@ export class SDKClient {
         this.operatorAccountId = clientMain.operatorAccountId ? clientMain.operatorAccountId.toString() : 'UNKNOWN';
 
         // clear and create metrics in registry
-        const metricHistogramName = 'rpc_relay_consensusnode_response';
-        register.removeSingleMetric(metricHistogramName);
-        this.consensusNodeClientHistorgram = new Histogram({
-            name: metricHistogramName,
+        const metricHistogramCost = 'rpc_relay_consensusnode_cost';
```

Comment:
Can we leave the name of the metric, name of the const is fine, but the name of the metric is important for the dashboards that we already have in place.

```suggestion
        const metricHistogramCost = 'rpc_relay_consensusnode_response';
```